sagemaker-featurestore/feature_store_object_detection_ground_truth.ipynb

```diff
@@ -0,0 +1,484 @@
+    "\n",
+    "### Library Dependencies:\n",
+    "* `sagemaker>=2.0.0`\n",
+    "* `numpy`\n",
+    "* `pandas`\n",
+    "* `boto3`\n",
+    "\n",
+    "### Data\n",
+    "This notebook uses a synthetic manifest file. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import json\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import sagemaker\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Set up"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sagemaker_session = sagemaker.Session()\n",
+    "s3_bucket_name = sagemaker_session.default_bucket() # This is the bucket for your Feature Store.\n",
+    "prefix = 'sagemaker-featurestore-demo'\n",
+    "role = sagemaker.get_execution_role()\n",
+    "region = sagemaker_session.boto_region_name"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Additional - Helper Method\n",
+    "Below is a method that you can use to get your manifest file from your S3 bucket into your current working directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def download_file_from_s3(bucket, path, filename):\n",
+    "    '''\n",
+    "    bucket: S3 bucket name\n",
+    "    path: path to file\n",
+    "    filename: the name of the file you are downloading\n",
+    "\n",
+    "    Download filename to your current directory.\n",
+    "    Return: None\n",
+    "    '''\n",
+    "    import os.path\n",
+    "    if not os.path.exists(filename):\n",
+    "        s3 = boto3.client('s3')\n",
+    "        s3.download_file(\n",
+    "            Bucket = bucket,\n",
+    "            Key = path,\n",
+    "            Filename = filename\n",
+    "        )\n",
+    "# Supply the path to your output.manifest file from your AWS Ground Truth Classification labelling job. \n",
+    "#download_file_from_s3(public_s3_bucket_name, path='PATH/TO/GROUND-TRUTH/CLASSIFICATION-JOB/', filename='output.manifest')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Prepare your manifest file for Feature Store. \n",
+    "\n",
+    "Below is a method that will parse your `output.manifest` file into a Pandas data frame for ingestion into your Feature Store. At this point, it is assumed that your manifest file is in your current working directory. "
```

Comment:
"Use the following method to parse your `output.manifest` file into a Pandas data frame for ingestion into your feature store. At this point, we assume that your manifest file is in your current working directory. "